library/psa_crypto.c

```diff
@@ -4695,6 +4695,129 @@ static void psa_des_set_key_parity( uint8_t *data, size_t data_size )
 }
 #endif /* MBEDTLS_PSA_BUILTIN_KEY_TYPE_DES */
 
```

Comment:
In https://github.com/ARMmbed/mbedtls/pull/5218 I wrote a script that can generate expected output for test cases.

Since you haven't pushed a new test function yet, I wrote my own. If you have a different test function in your working copy I or you can adapt my code.

Reply:
I was testing the implementation as described [here](https://github.com/ARMmbed/mbedtls/pull/5139#issuecomment-972778279).

Thanks I'll try to analyse and use your script.

Reply:
I've added ECC test cases, including a Weierstrass test case that requires redrawing.